conf/app.ini

```diff
@@ -74,6 +74,11 @@ ORG_PAGING_NUM = 50
 ; Number of repos that are showed in one page
 REPO_PAGING_NUM = 15
 
+[ui.meta]
+AUTHOR = Gitea - Git with a cup of tea
+DESCRIPTION = Gitea: Git with a cup of tea
```

Comment:
It seems DESCRIPTION's default value is `Gitea (Git with a cup of tea) is a painless self-hosted Git service written in Go` ?